wgpu-core/src/binding_model.rs

```diff
@@ -1164,6 +1164,14 @@ pub(crate) fn buffer_binding_type_bounds_check_alignment(
     }
 }
 
+#[derive(Debug)]
+pub(crate) struct BindGroupLateBufferBindingSize {
```

Comment:
nit: It's sorta weird to call this a `size` when it includes an index, too. 🤔

Reply:
It info now!